src/components/Floor/Floor.tsx

```diff
@@ -0,0 +1,34 @@
+import {SFloor, SFloorInner} from './styles';
+import {Button} from '@src/styles/Button/Button';
+import {FLOOR_NAME} from '@src/constants';
+
+export type FloorPropsType = {
+    floor: number
+    onStartLift: (floor: number) => void
+    isPressed: boolean
+}
+
+export const Floor = (props: FloorPropsType) => {
+
+    const onclickHandler = () => {
+        props.onStartLift(props.floor)
+    }
+
+    return (
+        <SFloor>
+            <SFloorInner>
+                {FLOOR_NAME}
```

Comment:
Константа используется только здесь, почему-бы сразу текст не написать?
```suggestion
                Этаж
```